config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :dish, only: [:show]
+  resources :dish_ingredients, only: [:create]
```

Comment:
These are great RESTful paths! If you want to handroll paths on an IC that's okay too.

Reply:
Just be sure to use `rails routes` to take a look at what controllers these default to directing to. Also, we usually use the plural in the resources syntax (i.e. `resources :dishes`) but that's very minor and we wouldn't penalize you for singular/plural here -- just sharing for knowledge of the conventions!